server/fishtest/rundb.py

```diff
@@ -1163,6 +1171,48 @@ def count_games(d):
 
         return ret
 
+    def check_results(self, run, run_id, task_id):
+        old = run["results"]
```

Comment:
I seem to delete the first comment instead of marking this resolved (which it seems I can't).
Well, all in all if this works let's move forward with it..